src/drivers/ulanding/ulanding.cpp

```diff
@@ -92,6 +149,16 @@ Radar	*g_dev;
 }
 
 Radar::Radar(const char *port) :
+#if defined(__PX4_POSIX)
+	VDev("Radar", RANGE_FINDER0_DEVICE_PATH),
```

Comment:
Then this can be only around only the VDev/CDev line instead of duplicating everything.